external/js/cothority/README.md

```diff
@@ -91,3 +93,45 @@ will be provided thanks to the [buffer package](https://www.npmjs.com/package/bu
 As this is a [polyfill](https://remysharp.com/2010/10/08/what-is-a-polyfill), please
 check that what you need is implemented or you will need to use a different approach. Of
 course for NodeJS, you will always get a [Buffer](https://nodejs.org/api/buffer.html).
+
+### Use a developpment version from an external app
+
+**Note**: Using the official command `yarn link` doesn't seem to work. We leave
+the steps at the end for informational purpose, but we found out that using symlink
+is simpler, clearer, and actually works.
+
+Build and Link the local `@dedis/cothority` module to a local app:
+
+```bash
+(cothority/external/js/cothority) $ npm run build
+$ ln -s /<cothority root path>/cothority/external/js/cothority/dist /<local app root path>/node_modules/@dedis/cothority
+```
+
+**For information purpose**: The "clean" way to symlink a local package is
+with the `yarn link` command. However it happens to link the root folder
+instead of the `root/dist` one. Hence this is not likely to work:
+
+1) Build a version
+
+```bash
+(cothority/external/js/cothority) $ npm run build
+```
+
+2) Create a link
+
+```bash
+(cothority/external/js/cothority/dist) $ yarn link
+```
+
+3) From the root folder of the external app, link the new package
+
+```bash
+(external_app) $ yarn link "@dedis/cothority"
+```
+
+4) If needed, install the new linked pachage
+
+```bash
+(external_app) $ npm install --save @dedis/cothority
```

Comment:
why using npm in that case ? I guess you mean `yarn add --save` ?

Reply:
Also I'm not sure we should encourage using yarn as cothority and kyber are using npm. Or we should switch to yarn everywhere but it seems confusing and frustrating to mix the usage of both.

Reply:
I totally agree! Yarn is used in the student_18_explorer project while cothority uses npm. Thats something we should agree on some guidelines.